pgt/src/PGT/Output/Test/Comments.hs

```diff
@@ -54,7 +54,7 @@ instance Render Comments where
 
 parse :: Parser Comments
 parse = do
-  comments <- NonEmpty.fromList <$> Text.many1' parseCommentLine
+  text <- NonEmpty.fromList <$> Text.many1' parseCommentLine
```

Comment:
__question:__ I noticed this still calls `parseCommentLine`, should that function be renamed too, or is it ok to leave it as-is since it will be used elsewhere to parse comment lines?